Connector/config.json

```diff
@@ -1,3 +1,5 @@
 {
-    "version": "1.1.7"
+    "version": "1.1.7",
+    "availableCurrenciesFile": "../.availableCurrencies.json",
+    "defaultPkgConfFile": "defaultConf.json"
```

Comment:
Is this necessary? We can call this `defaultConf.json` directly from the code instead of read it from `config.json`. This is because we only need the name of the file and this name should be the same in every API configuration.

however `availableCurrenciesFile` is a path so we want to keep it here in `config.json`

Reply:
Each package needs a `defaultConf.json` to work with, to keep consistency and do not have different file names in each package

Reply:
Fair enought